tests/vmi_multus_test.go

```diff
@@ -593,21 +579,41 @@ var _ = Describe("SRIOV", func() {
 
 	tests.BeforeAll(func() {
 		tests.BeforeTestCleanup()
-		// Create two sriov networks referring to the same resource name
+		// Extra cleaning of sriov networks
 		result := virtClient.RestClient().
+			Delete().
+			RequestURI(fmt.Sprintf(sriovNetworksEndpoint, "")).
+			Do()
+		Expect(result.Error()).NotTo(HaveOccurred())
+
+		// Create two sriov networks referring to the same resource name
+		result = virtClient.RestClient().
 			Post().
-			RequestURI(fmt.Sprintf(postUrl, tests.NamespaceTestDefault, "sriov")).
+			RequestURI(fmt.Sprintf(sriovNetworksEndpoint, "sriov")).
 			Body([]byte(fmt.Sprintf(sriovConfCRD, "sriov", tests.NamespaceTestDefault))).
 			Do()
 		Expect(result.Error()).NotTo(HaveOccurred())
 		result = virtClient.RestClient().
 			Post().
-			RequestURI(fmt.Sprintf(postUrl, tests.NamespaceTestDefault, "sriov2")).
+			RequestURI(fmt.Sprintf(sriovNetworksEndpoint, "sriov2")).
 			Body([]byte(fmt.Sprintf(sriovConfCRD, "sriov2", tests.NamespaceTestDefault))).
 			Do()
 		Expect(result.Error()).NotTo(HaveOccurred())
 	})
 
+	AfterEach(func() {
+		// Multus tests need to ensure that old VMIs are gone
```

Comment:
Otherwise what? How comes this hasn't been already?

Reply:
It was when multus + sriov were a single test suite. 
It probably got lost after they got split. If we have dangiling vms (and a lot of tests) we risk to finish the vfs.